cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -154,6 +155,7 @@
     private final SecretOperator secretOperations;
     private final PlatformFeaturesAvailability pfa;
     private final Optional<LabelSelector> kafkaSelector;
+    boolean usingJbodStorage = false;
```

Comment:
Just reviewing in Github, so it's hard to fully check but it looks like this field could be private

Reply:
Also, you do not need to explicitly set false here. Javac compiler sets it for you (default state)